setup.py

```diff
@@ -57,8 +57,7 @@
     "bigframes": ["bigframes >= 1.17.0"],
     "geopandas": ["geopandas >= 1.0.1"],
     "spanner-graph-notebook": [
-        "spanner-graph-notebook >= 1.1.1, <=1.1.1",
-        "networkx",
+        "spanner-graph-notebook >= 1.1.3",
```

Comment:
Should we put an upper bound on this? Or are all the APIs we're using now pretty stable?

If need be, I'd be OK putting an upper bound if/when it becomes a problem. Ideally we'd install the spanner-graph-notebook package from the GitHub repo here in the prerelease_deps session https://github.com/googleapis/python-bigquery-magics/blob/main/noxfile.py#L431 similar to what we do here: https://github.com/googleapis/python-bigquery-pandas/blob/7bac1c7439dcdce09edac6461a5127321451daee/noxfile.py#L345-L351